examples/cublas/BUILD.bazel

```diff
@@ -2,5 +2,5 @@
 cc_binary(
     name = "main",
     srcs = ["cublas.cpp"],
-    deps = ["@local_cuda//:cublas"],
```

Comment:
I haven't come up with a good way of avoiding knowing the resolved toolchain repo name here to find cublas.

Reply:
`@local_cuda//:cuda_runtime` is also missing, so in #66 the basic CI failed. And it should be resolved before this one.